src/client/java/teammates/client/scripts/scalabilitytests/InstructorCourseEnrollPageScaleTestDataGenerator.java

```diff
@@ -0,0 +1,51 @@
+package teammates.client.scripts.scalabilitytests;
+
+import org.kohsuke.randname.RandomNameGenerator;
+import teammates.test.driver.FileHelper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Generates test data for InstructorCourseEnrollPageScaleTest.
+ */
+public final class InstructorCourseEnrollPageScaleTestDataGenerator {
```

Comment:
Perhaps there is no need to mention Scalability here? This class simply generates enroll strings; what they are used for is not this class' business.